### PR TITLE
storage: implement shrink partition action in reclaim dialog

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Review by adamkankovsky — COMMENTED** [review]
I tried resizing one volume and deleting the Btrfs volume in the ReclaimSpaceOptional test dialog and got a critical error:
image

**Comment by KKoukiou** [comment]
> I tried resizing one volume and deleting the Btrfs volume in the ReclaimSpaceOptional test dialog and got a critical error: image Theoretically there could be a problem in resizing to the whole value. E.g. 1GB

I added a Math.round to ensure that this is never float value.

File: test/check-storage-reclaim [hunk 1]
```diff
@@ -216,6 +216,30 @@ class TestStorageUseFreeSpaceScenarioExistingSystemFedora(anacondalib.VirtInstal
         i.reach(i.steps.REVIEW)
         r.check_affected_system("Fedora Linux", [("vda4", ["home", "root", "var"])])
 
+@nondestructive
+class TestStorageUseFreeSpaceScenarioExistingSystemDebian(anacondalib.VirtInstallMachineCase, StorageCase):
```
Comment (rvykydal) [inline test/check-storage-reclaim]: This may go in a separate commit but I don't insist, just making sure it is really intended to go in.

**Review by rvykydal — APPROVED** [review]
Looks good to me, thank you!
